src/screens/TokenDetail.js

```diff
@@ -286,6 +353,50 @@ class TokenDetail extends React.Component {
       );
     }
 
+    const renderIcons = () => {
+      if (this.state.tokenUnknown) {
+        return (
+          <div>
+            <i className="fa fa-warning text-warning ml-3" title="Token is not registered"></i>
+            <i className="fa fa-check pointer ml-3" title="Register token" onClick={this.registerClicked}></i>
```

Comment:
I don't like the "correct" symbol next to the warning symbol. Maybe we should open an explanation popup when the user click on the warning. And there may have a register button. What do you think?

Reply:
All screens have changed a lot. Can you check again, please?